src/ngx_http_lua_socket_tcp.c

```diff
@@ -2963,6 +2981,225 @@ ngx_http_lua_socket_tcp_send(lua_State *L)
     return lua_yield(L, 0);
 }
 
```

Comment:
The nginx coding style requires the use of 2 blank lines to separate function definitions. See the guide mentioned above.